### PR TITLE
NO-TICKET - Debug view visibility persistence

### PR DESCRIPTION
**Pull request by 1sakov**

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]
yuhoo! thx

File: app/src/main/java/org/p2p/wallet/root/RootActivity.kt [hunk 1]
```diff
@@ -105,19 +110,24 @@ class RootActivity : BaseMvpActivity<RootContract.View, RootContract.Presenter>(
     @SuppressLint("SetTextI18n")
     private fun initializeDebugDrawer() {
         val devView = findViewById<TextView>(R.id.developTextView)
+        val isDebugHidden = sharedPreferences.getBoolean(KEY_DEBUG_HIDDEN, false)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/root/RootActivity.kt:113]:
```suggestion
        val isDebugPanelHidden = sharedPreferences.getBoolean(KEY_DEBUG_HIDDEN, false)
```

we can't hide debug itsself :)

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/root/RootActivity.kt (hunk 1 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/root/RootActivity.kt:113]: but it's not panel, it's debug view

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/root/RootActivity.kt:113]: I can rename to `isDebugViewHidden` if you don't mind @gslevinkov

**Review by davran1001 — APPROVED** [review]

**Review by kuznetsov — APPROVED** [review]

**Pull request merged**